docs/quickstart/docker.md

```diff
@@ -90,12 +97,14 @@ ENV SPARK_VERSION=2.3.3
 ENV HADOOP_VERSION=2.7
 
 # support Kerberos certification
-RUN install -yq curl unzip wget grep sed vim krb5-user libpam-krb5 && apt-get clean
+RUN export DEBIAN_FRONTEND=noninteractive && apt-get update && apt-get install -yq krb5-user libpam-krb5 && apt-get clean
+
+RUN apt-get update && apt-get install -y curl unzip wget grep sed vim tzdata
```

Comment:
do this before the krb5 one?
also, at the end should add ` && apt-get clean`

Reply:
1. They don't have a sequence, I just want to separate the installation of these two components.
2. Ok, let me add `&& apt-get clean`